ci/ray_ci/tester_container.py

```diff
@@ -41,6 +47,19 @@ def __init__(
         if not skip_ray_installation:
             self.install_ray(build_type)
 
+    def _create_bazel_log_mount(self) -> Tuple[str, str]:
+        """
+        Create a temporary directory in the current container to store bazel event logs
+        produced by the test runs. We do this by using the artifact mount directory from
+        the host machine as a shared directory between all containers.
+        """
+        tmp_dir = "".join(random.choice(string.ascii_lowercase) for _ in range(5))
+        artifact_host, artifact_container = self.get_artifact_mount()
+        bazel_log_dir_host = os.path.join(artifact_host, tmp_dir)
+        bazel_log_dir_container = os.path.join(artifact_container, tmp_dir)
+        os.mkdir(bazel_log_dir_container)
```

Comment:
this feel very similar with doing `tempfile.TemporaryDirectory` though, should I use that instead and piggy back on python to clean up the directory for me?

Reply:
up to you. I think either is fine. using `tempfile.TemporaryDirectory` does not give us a lot of benefit here.

I personally feel that not using `tempfile` is slightly more reasonable. `tempfile.TemporaryDirectory` is supposed to be "private", meaning it is set up in a (secure) way that only the owner can access it, and it should not be mounted from one namespace/container into another.